addons/web/static/tests/views/list_view_tests.js

```diff
@@ -4012,8 +4012,9 @@ QUnit.module("Views", (hooks) => {
             );
             assert.strictEqual(
                 target.querySelectorAll("tfoot td")[1].textContent,
-                "—",
-                "aggregates monetary should never work if different currencies are used"
+                "",
+                "monetary aggregation should only be attempted with an active aggregation function" +
+                " when using different currencies"
             );
```

Comment:
```suggestion
            );
            assert.strictEqual(
                target.querySelectorAll("tfoot td")[2].textContent,
                "",
                "monetary aggregation should only be attempted with an active aggregation function" +
                " when using different currencies"
            );
```
should do the work to test your fix